index.html

```diff
@@ -1,57 +1,213 @@
+            </p>
+          </div>
+          <ul class="pro-tag">
+            <li class="pro-h">html</li>
+            <li class="pro-h">bootstrap</li>
+            <li class="pro-h">Ruby</li>
+          </ul>
+             <button class="see-proj" type="button">See Project</button>
+        </div>
+        <div class="professional-work">
+          <div class="pro-art">
+           <h1 class="art-print">Profesional Art Printing Data</h1> 
+            <p class="daily-selection">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <ul class="pro-tag">
+            <li class="pro-h">html</li>
+            <li class="pro-h">bootstrap</li>
+            <li class="pro-h">Ruby</li>
+          </ul>
+             <button class="see-proj" type="button">See Project</button>
+        </div>
+        <div class="professional-work">
+          <div class="pro-art">
+           <h1 class="art-print">Profesional Art Printing Data</h1> 
+            <p class="daily-selection">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <ul class="pro-tag">
+            <li class="pro-h">html</li>
+            <li class="pro-h">bootstrap</li>
+            <li class="pro-h">Ruby</li>
+          </ul>
+             <button class="see-proj" type="button">See Project</button>
+        </div>
+        <div class="professional-work">
+          <div class="pro-art">
+           <h1 class="art-print">Profesional Art Printing Data</h1> 
+            <p class="daily-selection">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <ul class="pro-tag">
+            <li class="pro-h">html</li>
+            <li class="pro-h">bootstrap</li>
+            <li class="pro-h">Ruby</li>
+          </ul>
+             <button class="see-proj" type="button">See Project</button>
+        </div>
+        <div class="professional-work">
+          <div class="pro-art">
+           <h1 class="art-print">Profesional Art Printing Data</h1> 
+            <p class="daily-selection">A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <ul class="pro-tag">
+            <li class="pro-h">html</li>
+            <li class="pro-h">bootstrap</li>
+            <li class="pro-h">Ruby</li>
+          </ul>
+             <button class="see-proj" type="button">See Project</button>
+        </div>
+        <div class="professional-work">
+          <div class="pro-art">
+           <h1 class="art-print">Profesional Art Printing Data</h1> 
```

Comment:
- [ ] Kindly change the `<h1>` tag to any of `<h2>, <h3>, ..., <h6>` tag.
`<h1>` tags tell search engine bots and web users what a page is about. A webpage's `<h1>` is the most important heading and should accurately summarize the contents of that page.
You can read more [here](https://www.semrush.com/blog/h1-tag/#:~:text=H1%20tags%20tell%20search%20engine,the%20contents%20of%20that%20page.).